db/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/db/river/LagreImRiver.kt

```diff
@@ -79,7 +69,7 @@ class LagreImRiver(
                 data
                     .plus(
                         mapOf(
-                            Key.ER_DUPLIKAT_IM to erDuplikat.toJson(Boolean.serializer()),
+                            Key.ER_DUPLIKAT_IM to false.toJson(Boolean.serializer()),
```

Comment:
Gjør dette for å opprettholde eksisterende flyt av meldinger. Vi bør vurdere om vi i det hele tatt trenger å lagre Inntektsmelding-klassen når vi allerede lagrer InntektsmeldingSkjema-klassen.

Reply:
Kan vi legge inn en kommentar her som sier at dette er midlertidig?